.docs/content/1.concepts/3.execute-tests.md

```diff
@@ -0,0 +1,88 @@
+# How to execute tests in ArmoniK Core?
+
+## Unit tests independent of deployment
+For this type of tests, you can directly execute it. The first option is to run the tests using your code editor. The second is to open a terminal, go to the folder where the test project is located and do:
+```
+dotnet test
+```
+## Unit tests that need partial deployment
+First, you should do the needed partial deployment that deploys the container running needed service.
+For example, for the case of ArmoniK.Core.Adapters.RabbitMQ.Tests, execute:
+```
+just queue=rabbitmq deployTargetQueue
+```
+For ArmoniK.Core.Adapters.LocalStorage.Tests deploy with local object storage:
+```
+just object=local deployTargetObject
+```
+And so you can execute the corresponding tests:
+- via your code editor
+- or in your command line, go to the folder where the test project is located and do:
+```
+dotnet test
+```
+
+## Integration tests
+
+HtcMock, Stream and Bench are integration tests that need a Core deployment. To be able to run these tests, you need first to deploy ArmoniK Core with the appropriate values for the parameter `worker`: htcmock, stream or bench. If you don't specify any value for this parameter, it's set to htcmock by default. Don't forget to use the recipe to build the images for your deployments if they do not exist.
+
+### Htcmock
+The default deployment is done with htcmock worker:
+```
+just deploy
```

Comment:
we need somehow to explain that we can build the images locally before deploying them. We have two options:
- Use existing images already built (locally with default tag or from pipeline with a release or branch tag)
- Build images locally and deploy with these images : `build-deploy` target
It could be explained in its own section